src/components/Shared/Header/ListHeader.test.tsx

```diff
@@ -112,9 +111,9 @@ const router = {
   push,
 };
 
-beforeEach(() => {
-  (useAccountListId as jest.Mock).mockReturnValue(router);
-});
+// beforeEach(() => {
+//   (useAccountListId as jest.Mock).mockReturnValue(router);
+// });
```

Comment:
You can remove this code instead of commenting it out. @canac maybe you should have a quick look at this PR as well. I think it looks good.